### DISCUSSION TITLE
Is there any way to use "github_integration" resource without a pipeline?

### DISCUSSION THREAD

**Discussion by shekelboi** [davidji99/terraform-provider-herokux#398] [Q&A]
I want to use "github_integration" without creating a pipeline. The same thing can be done on Heroku's website. When I create the GitHub integration there and I use terraform import, I can import the integration, without creating a pipeline, however I cannot just create a "github_integration" without a pipeline (I need to provide the name of the repository from somewhere).

Is it possible to achieve this? Why is it not possible to set the "repository" or "repository_id" attributes within the resource?

Thanks in advance.

**Comment by davidji99** [comment]
Hi,

Which resource are you referring to: `herokux_app_github_integration` or `herokux_pipeline_github_integration`?

Can you also paste me your configuration?

Generally, actions you can do via the UI sometimes don't have equivalent functionality in the non-public Heroku APIs used by this provider.

**Reply by shekelboi** [reply]
Hello,

Thank you for your help and sorry for my late response. I was referring to `herokux_app_github_integration`, I cannot create it without pipeline integration, since it is not possible to refer to a GitHub repository without the `herokux_pipeline_github_integration`, at least that's what I saw, based on the documentation, since only `herokux_pipeline_github_integration` the `org_repo` attribute.

Is there any way to define the repostiory without having a pipeline?

Thanks in advance.
